### ISSUE TITLE
Taking hours to download nt database through kraken

### ISSUE THREAD

**Issue by Lamm-a** [DerrickWood/kraken2#355]
I am trying to build the `kraken nt` database with 5 threads and 20gb of memory but it is taking over 5 hours. Is this normal?

I am unsure of the up and down speeds but I am working on a HPC so it is unlikely that is the issue.

`kraken2-build --use-ftp --download-library nt --db kraken2DB`

**Comment by AGalanis97** [comment]
Yes! I build the nt usually in 16-24 hrs including all downloads (taxonomy and nt). The nt is about 200GB so depending on your connection it may take a while. Also in my experience ftp is a bit slower so be patient! :)

**Comment by Lamm-a** [comment]
Okay thank you :) What does Kraken do different with the database as downloading the raw NT database from NCBI taken no where near that long?

**Comment by AGalanis97** [comment]
I'm unsure about this, because in my case regardless of whether I download nt with or without kraken it takes a lot of time. But I think that if you can download the nt by yourself you can just proceed to the building step.. I wouldn't advise so since I haven't tried it and I'm not sure if kraken downloads and reformats the FASTA file somehow.. Probably @jenniferlu717 can help answer that :)

**Comment by NortheastPengYuyan** [comment]
@Lamm-a HI，You can modify download_genomic_library.sh, change "wget -q" to "wget -c", or if you have the latest downloaded nt.gz file locally, you can copy it to kraken2DB/library/nt/ and run kraken2- build --use-ftp --download-library nt --db kraken2DB


![shell_16511064181097](https://user-images.githubusercontent.com/90658750/165653744-fcd6a089-e4a1-4926-a045-01ba60ca78c0.png)
